pkg/extensions/unpack.go

```diff
@@ -81,7 +84,7 @@ func findRegistryRoot(fs afero.Fs, dir string) string {
 	return dir
 }
 
-func doUnpack(fs afero.Fs, root string) (string, error) {
+func doUnpack(fs afero.Fs, root string) (string, error) { // nolint:gocyclo
```

Comment:
I recommend leaving a small comment with a bit of context when overriding the linter, even if it's just something like "this function is more readable with slightly higher cyclomatic complexity"